tab-command/src/service/main/list_tabs.rs

```diff
@@ -53,10 +60,27 @@ impl MainListTabsService {
         let target_len = len + 4;
 
         println!("Available tabs:");
+        let cwd: String = env::current_dir()
```

Comment:
You could use https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.starts_with to do this path test.  It'd simplify this construction & solve some tricky path issues.